packages/api/src/controllers/asset.ts

```diff
@@ -169,6 +169,30 @@ async function getActiveObjectStore(id: string) {
   return os;
 }
 
+export type StaticPlaybackInfo = {
+  playbackUrl: string;
+  size: number;
+  height: number;
+  width: number;
+  bitrate: number;
+};
+
+export function getStaticPlaybackInfo(
+  {}: Request["config"],
+  asset: WithID<Asset>,
+  os: ObjectStore
+): StaticPlaybackInfo[] {
+  return (asset.files ?? [])
+    .filter((f) => f.type === "static_transcoded_mp4")
+    .map((f) => ({
+      playbackUrl: pathJoin(os.publicUrl, asset.playbackId, f.path),
+      size: f.spec?.size ?? null,
+      height: f.spec?.height ?? null,
+      width: f.spec?.width ?? null,
+      bitrate: f.spec?.bitrate ?? null,
```

Comment:
Are these values optional? If they are only not present during the intermediate versions we made during testing, I think we should remove this coalescing logic here. It will only make ourselves wonder in 6 months "omg are these not always present?" hahaha

Reply:
They are not optional, just being afraid while on staging that something wasn't coming in! Removed :)